widgets/common/src/LabelsInput.jsx

```diff
@@ -0,0 +1,126 @@
+import AddButton from './labels/AddButton';
+import LabelsList, { LabelsPropType } from './labels/LabelsList';
+import KeyDropdown from './labels/KeyDropdown';
+import ValueDropdown from './labels/ValueDropdown';
+import { addSearchToUrl } from './labels/common';
+
+export default function LabelsInput({ initialValue, onChange, toolbox }) {
```

Comment:
Can we change `initialValue` to `initialLabels`? This way the variable name provides a bit more contextual information (we know the values are labels, not arbitrary "values")

```suggestion
export default function LabelsInput({ initialLabels, onChange, toolbox }) {
```

Reply:
Yep, changed.